SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/AnimateVisitor.h

```diff
@@ -46,20 +42,17 @@ class SOFA_SIMULATION_CORE_API AnimateVisitor : public Visitor
 protected :
     SReal dt;
     bool firstNodeVisited;
+
 public:
-    AnimateVisitor(const core::ExecParams* params = core::execparams::defaultInstance());
     AnimateVisitor(const core::ExecParams* params, SReal dt);
 
     void setDt(SReal v) { dt = v; }
     SReal getDt() const { return dt; }
 
     virtual void processCollisionPipeline(simulation::Node* node, core::collision::Pipeline* obj);
-    virtual void processBehaviorModel(simulation::Node* node, core::BehaviorModel* obj);
```

Comment:
was not used at all? Do you think it could be used in a plugin? 
If yes, maybe put as deprecated with a message telling what to use instead?

Reply:
I restored them and marked them as deprecated. I did not notice any warning due to this deprecation. I think they are really not used.